pkgs/build-support/rust/fetch-cargo-deps

```diff
@@ -50,7 +50,18 @@ EOF
     export CARGO_HOME=$out
     cd $src
 
-    if [[ ! -f Cargo.lock ]]; then
+    find_upwards() {
+        [[ "$(realpath "$1")" == "/" ]] && return 1
+        if [ -f "$1/$2" ]; then
+            echo "$1/$2"
+        else
+            func "../$1" "$2"
```

Comment:
this should be `find_upwards`

Reply:
whops,... copy pasted in my test script and forgot to rename :-/